### PR TITLE
Regression Tests: Test helper events

### PR DESCRIPTION
**Pull request by gitjain**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.visualtest.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+commands.push(mouseClickScreenshot);
+
+componentIds.push({
+  selector: '.' + 'DocumentCard',
+  fileName: 'documentCard',
+  commands: commands
+});
+
+function testRunner() {
+  componentIds.forEach(element => {
```
Comment (joschect) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.visualtest.ts]: I think you might be able to move test runner to your helper file.

Comment (gitjain) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.visualtest.ts]: Sure. Will update

**Comment by micahgodbolt** [comment]
@joschect what's up with that big shrinkwrap file change? Is that appropriate?

**Comment by joschect** [comment]
If rush generate was run, then yes, but I don't think that would be needed. We should revert that.

**Comment by micahgodbolt** [comment]
One other request. @gitjain can we move the button tests into the individual folders? That button root level is really messy.

**Comment by micahgodbolt** [comment]
closing this as #1644 is doing the same

**Pull request closed without merging**